UKHomeOffice/sgmr-service
# SPCR-164 Update eslint package in preparation for other package updates

## Description
- we need to update babel-eslint-parser as babel-eslint is no longer supported
- babel-eslint-parser requires eslint 7.x+ 

This ticket
- updates eslint to the latest version
- adds babel-eslint-parser
- resolves a few linting errors that were identified

## To test
- remove your package-lock.json and nodemodules
- `npm i`
- `npm run lint`
- > there should be no linting errors
- `npm start`
- > the app should load

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
